ui/src/shared/apis/annotation.ts

```diff
@@ -41,22 +37,32 @@ export const createAnnotation = async (
 export const getAnnotations = async (
   url: string,
   since: number,
-  until: number
+  until: number | null,
+  tagFilters: TagFilter[]
 ) => {
-  const {data} = await AJAX({
-    method: 'GET',
-    url,
-    params: {since: msToRFCString(since), until: msToRFCString(until)},
-  })
+  let paramedUrl = `${url}?since=${encodeURIComponent(msToRFCString(since))}`
```

Comment:
thought about it, and decided it wasn't worth the hassle :) I'm also not sure there's a consistent way we should serialize array query parameters. Seems like it could be

```
?foo=a,b,c
```

in some cases and 

```
?foo=a&foo=b&foo=c
```

in others.